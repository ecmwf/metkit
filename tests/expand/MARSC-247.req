 retrieve,
    class=rd,
    expver=aaaa,
    target=data.reference