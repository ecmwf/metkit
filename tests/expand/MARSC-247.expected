RETRIEVE,
    CLASS      = RD,
    TYPE       = AN,
    STREAM     = OPER,
    EXPVER     = aaaa,
    REPRES     = SH,
    LEVTYPE    = PL,
    LEVELIST   = 1000/850/700/500/400/300,
    PARAM      = 129,
    DATE       = 20250413,
    TIME       = 1200,
    STEP       = 00,
    DOMAIN     = G,
    TARGET     = data.reference 